### ISSUE TITLE
Edge Type in Traversal

### ISSUE THREAD

**Issue by francnuec** [arangodb/arangodb#2233]
Do you plan to support specifying the type of edge collection to consider in traversals? Take this example from the docs:
```
FOR v, e, p IN 1..5 OUTBOUND 'circles/A' GRAPH 'traversalGraph'
  FILTER p.edges[*].theTruth ANY == true
  RETURN p
```

How do I specify collection type/name for "e" and avoid all edges on my vertices being considered in the operation.

**Comment by akki** [comment]
You can specify the desired edge collection in the query itself instead of `GRAPH 'traversalgraph'` like [this](https://docs.arangodb.com/3.0.10/AQL/Graphs/ShortestPath.html#working-with-collection-sets).
Is this what you wanted, sorry if I misunderstood your issue.

**Comment by dothebart** [comment]
Hi, 
This issue was fixed with 3.1.8; Meanwhile 3.1.9 is available. 
Sorry for this slipping through - Closing.

**Comment by Simran-B** [comment]
Kaveh's referenced bugfix does not seem to be related to this issue. I think the actual question was if/how it's possible to limit the traversal of a **named** graph to certain edge collections.
